services/src/main/java/io/scalecube/services/ServiceScanner.java

```diff
@@ -21,8 +24,16 @@ private ServiceScanner() {
    */
   public static List<ServiceRegistration> scanServiceInfo(ServiceInfo serviceInfo) {
     return Arrays.stream(serviceInfo.serviceInstance().getClass().getInterfaces())
-        .map(serviceInterface -> new InterfaceInfo(serviceInterface, serviceInfo.tags()))
-        .filter(interfaceInfo -> interfaceInfo.serviceInterface.isAnnotationPresent(Service.class))
+        .filter(serviceInterface -> serviceInterface.isAnnotationPresent(Service.class))
+        .map(
+            serviceInterface -> {
+              Map<String, String> userTags = serviceInfo.tags();
```

Comment:
In contrast to `apiTags` where it's clear what is it, the `userTags` var name sounds confusing. Lets rename it to `clientTags` or `serviceTags` or `programTags`. Because we have to designate that this is _programmatic_ tags coming from ServiceInfo form the client.